src/actions.cpp

```diff
@@ -73,7 +73,7 @@ namespace
       //! Read the timestamp from a broken-down time in buffer \em tm.
       int read(struct tm *tm);
       //! Set the timestamp of a file
-      int touch(const std::string& path) const;
+      EXV_WARN_UNUSED_RESULT int touch(const std::string& path) const;
```

Comment:
Currently some CI jobs are failing because of this change:

```bash
/home/travis/build/Exiv2/exiv2/src/actions.cpp:820:34: error: ignoring return value of ‘int {anonymous}::Timestamp::touch(const string&) const’, declared with attribute warn_unused_result [-Werror=unused-result]
                 ts.touch(newPath);
```

If you want to proceed with the change, you will need to check the return value in all the usages of `touch()`

Reply:
Fixed.